views/confirmation.html

```diff
@@ -44,7 +44,7 @@ <h3 class="govuk-heading-s">What happens if the application is accepted</h3>
       <p class="govuk-body">The overseas entity and its beneficial owners will be added to the Register of Overseas Entities.</p>
 
       <h3 class="govuk-heading-s">What happens if the application is rejected</h3>
-      <p class="govuk-body">If it's rejected, we'll email you at <strong>{{ userEmail }}</strong> to explain what to do next. We'll also refund the £{{OE_CONFIGS.PAYMENT_FEE}} registration fee.</p>
+      <p class="govuk-body">If it's rejected, we'll email you at <strong>{{ userEmail }}</strong> to explain what to do next. We'll also refund the £{{ paymentFee }} {{ paymentProduct }} fee.</p>
```

Comment:
So it looks like this view is being reused for the UPDATE journey but a certain amount of text (above) is specific to registration - e.g. line 39 "We'll email you to let you know if your application to register an overseas entity has been accepted or rejected".

Is this correct/intended?

Reply:
The content is temporary / not finalised but looks like we have missed a mention of register which we are replacing.

Reply:
fixed